src/models.py

```diff
@@ -0,0 +1,36 @@
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+
+import auth_util
+
+db = SQLAlchemy()
+migrate = Migrate()
+
+class User(db.Model):
+    user_id = db.Column(db.Integer, primary_key=True)
+    email = db.Column(db.String(120), unique=True)
+    hashed_pw = db.Column(db.String(120))
+    email_verified = db.Column(db.Boolean())
+    email_verification_secret = db.Column(db.String(64))
+
```

Comment:
Let's go ahead and add the `is_destroyed` bool column to make the model soft-destroyable.
```
is_destroyed = db.Column(db.Boolean, server_default='f')
```